tests/integration/tests/tests.yaml

```diff
@@ -0,0 +1,17 @@
+gate_bundles:
+  - jammy
```

Comment:
Again, should we add focal, since most of the production controllers run on focal.